packages/firebase_admob/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,8 +8,10 @@
 
```

Comment:
A new test target was created, so the changes to this file need to be checked in.